cvmfs/server/cvmfs_server_apache.sh

```diff
@@ -271,6 +271,7 @@ Alias /cvmfs/${name} ${storage_dir}
     </FilesMatch>
 
     Header unset Last-Modified
+    RequestHeader unset If-Modified-Since
```

Comment:
I think it would be good to add a comment here, explaining that this is a workaround.

Reply:
Ok I added one for both Last-Modified and If-Modified-Since